README.md

```diff
@@ -24,9 +28,21 @@ To use another Apache installation, specify a path to the right apxs binary:
 
     ./configure --with-apxs=/foo/bar/apxs
 
-If you are compiling the module from a Git checkout, you must have `automake`,
-`autoconf`, and `libtool` installed and you must run `./bootstrap` before
-running `configure`.
+### From a GitHub "Source Code" Archive / Git Repo Clone (Achtung!) ###
+
+**NOTE:** These instructions are for installation from the GitHub "Source Code" archives also available on the [Releases](releases) page (e.g. `X.Y.Z.zip` or `X.Y.Z.tar.gz`), as well as installation directly from a clone of the Git repo. Installation from these sources are possible, but will present challenges to users not comfortable with manual dependency resolution.
+
+1. Ensure the build tools `automake`, `autoconf` and `libtool` are installed.
+1. Extract the archive and switch into the directory containing the extracted source.
```

Comment:
I'd suggest incrementing the numbers here so that it is more readable as plain text.